ExtraDry/ExtraDry.Core/Slug.cs

```diff
@@ -211,17 +222,33 @@ private static string ToSlugInternal(string name, bool lowercase = true, string
     private static string ToUnique(IEnumerable<string> existing, string stem)
     {
         var candidate = stem;
+        var hasValidStemCondition = stem.Any(c => c != '-');  // If the name is empty or is composed of only invalid characters, generate a random slug replacement rather than appending.
```

Comment:
Is `stem.Any(c => c != '-')` the same as `stem.Any(IsAsciiLetterOrDigit)`?  I had to read it 3 times and I'm only pretty sure they're the same.

Reply:
In this case it is, you're right that's much more legible. I've also updated the comment to reflect the change to the code.